.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  bump:
+    if: "!contains(github.event.head_commit.message, '[version bump]')"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: main
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+          registry-url: 'https://repo.backbase.com/artifactory/api/npm/npm-backbase/'
+          scope: '@backbase' 
+      - name: Setup git user to "GSA Bot"
+        run: git config user.email "actions@github.com" && git config user.name "GitHub Actions"
+      # - name: Release
+      #   uses: ./.github/actions/bump-version
+      #   with:
+      #     github_token: ${{ secrets.GITHUB_TOKEN }}
+      #     npm_token: ${{ secrets.NPM_TOKEN }}
+      - name: 'Extract project from tag'
+        id: set-project-from-tag
+        run: |
+          module=$(echo "${{ github.event.release.tag_name }}" | awk -F"-v" '{print $1}')
+          version=$(echo "${{ github.event.release.tag_name }}" | awk -F"-v" '{print $2}')
```

Comment:
```suggestion
          module=$(echo "${{ github.event.release.tag_name }}" | awk -F"@v" '{print $1}')
          version=$(echo "${{ github.event.release.tag_name }}" | awk -F"@v" '{print $2}')
```

Reply:
As the master config file had -v, I had kept it for testing purpose. Changed it to @v for the final PR